### PR TITLE
[WIP] making python3 compatible and maybe README suggestions

### PR DESCRIPTION
**Pull request by bannanc**
I made some changes to get the script python3 compatible. 

I added my example, without any documentation yet, but I think it would be useful for you to have something people (me/other undergrads) run to make sure things are installed and working on their computer even if you don't have complete tests to run.

### CODE REVIEW

File: confs2psi.py [hunk 1]
```diff
@@ -79,14 +79,14 @@ def confs2psi(insdf, method, basis, spe=False, memory=None):
 
     """
     wdir = os.getcwd()
-
```
Comment (bannanc) [inline confs2psi.py]: Just as a for the record, its good practice to have your file editor automatically remove white space at the end of files. Here's the line for a vimrc if you want it
```
autocmd FileType c,cpp,java,php,python autocmd BufWritePre <buffer> %s/\s\+$//e
```

**Comment by vtlim** [comment]
Hmm, can you try to see how your changes are consistent with the later commits? I don't want to revert to older code if this merge might do that.

Also - I already have examples using my naming convention I am planning to upload so you don't have to worry about that! Thanks though!

**Comment by bannanc** [comment]
The example was mostly just so I could learn how to use the code, I will remove it from the PR, I'm checking if I can use the code currently in the master branch.

File: confs2psi.py (hunk 1 above)
Comment (vtlim) [reply confs2psi.py]: Aghhh I have that in my vimrc but looks like it got a line break in there somehow. Should be fixed now, thanks.

**Comment by vtlim** [comment]
Ah, okay, I hadn't fixed the smi2confs file yet since mostly have been working with existing files and not starting from scratch (smiles strings). Everything looks good to go!

**Pull request merged**